client/src/main/java/tds/assessment/Item.java

```diff
@@ -24,7 +24,7 @@
     private boolean fieldTest;
     private boolean required;
     private String strand;
-    private List<ItemProperty> itemProperties;
+    private List<ItemProperty> itemProperties = new ArrayList<>();
```

Comment:
No issue with this, but I know that in other model classes such as Assessment and Segment, we initialize the empty array in the getter. Don't feel very strongly about it in either direction, but if we care about consistency this can follow that pattern.